### PR TITLE
osdocs8336: adding cgroup v2 default info to rn

### PR DESCRIPTION
**Pull request by DCChadwick**
Version(s): Enterprise 4.14

Issue: [OSDOCS-8336](https://issues.redhat.com//browse/OSDOCS-8336)

Link to docs preview: https://66977--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-nodes-cgroupv2-default

QE review:
- [x] QE has approved this change.

Additional information: cgroup v2 is now default, this info needs to be added to the release notes for OS 4.14

### CODE REVIEW

**Comment by DCChadwick** [comment]
@sunilcio & @jfrancoa This is ready for QE review.

**Comment by kalexand-rh** [comment]
> @sunilcio & @jfrancoa This is ready for QE review.

Tomorrow is the last day that we're promising to merge changes to the 4.14 docs before GA, so please confirm it as soon as you can.

**Review by jfrancoa — APPROVED** [review]
lgtm

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -1089,6 +1089,13 @@ With this release, specifying an unhealthy pod eviction policy for pod disruptio
 
 For more information, see xref:../nodes/pods/nodes-pods-configuring.adoc#pod-disruption-eviction-policy_nodes-pods-configuring[Specifying the eviction policy for unhealthy pods].
 
+[id="ocp-4-14-nodes-cgroupv2-default"]
+==== Linux Control Groups version 2 is now default
+
+Beginning with {product-title} 4.14, new installs will use Control Groups version 2 by default, also known as cgroup v2, cgroup2, or cgroupsv2. This enhancement includes many bug fixes, performance improvements, and the ability to integrate with new features. cgroup v1 is still used in upgraded clusters that have install born dates prior to {product-title} 4.14. cgroup v1 can still be used by changing the CgroupMode of the Node Config object to 'v1'.
```
Comment (adellape) [inline release_notes/ocp-4-14-release-notes.adoc:1095]:
```suggestion
Beginning with {product-title} 4.14, new installs use Control Groups version 2 by default, also known as cgroup v2, cgroup2, or cgroupsv2. This enhancement includes many bug fixes, performance improvements, and the ability to integrate with new features. cgroup v1 is still used in upgraded clusters that have install born dates prior to {product-title} 4.14. cgroup v1 can still be used by changing the CgroupMode of the Node Config object to 'v1'.
```

Comment (adellape) [inline release_notes/ocp-4-14-release-notes.adoc:1095]: Can "install born dates" just be "installation dates" or "initial installation dates"? Or is that a specific term we need to use.
```suggestion
Beginning with {product-title} 4.14, new installs will use Control Groups version 2 by default, also known as cgroup v2, cgroup2, or cgroupsv2. This enhancement includes many bug fixes, performance improvements, and the ability to integrate with new features. cgroup v1 is still used in upgraded clusters that have initial installation dates prior to {product-title} 4.14. cgroup v1 can still be used by changing the CgroupMode of the Node Config object to 'v1'.
```

Comment (adellape) [inline release_notes/ocp-4-14-release-notes.adoc:1095]: From looking up the `cgroupMode` field, it seems like "the Node Config object" is just a `Node` object, as far as how docs typically refer to them. Also a few more related markup:
```suggestion
Beginning with {product-title} 4.14, new installs will use Control Groups version 2 by default, also known as cgroup v2, cgroup2, or cgroupsv2. This enhancement includes many bug fixes, performance improvements, and the ability to integrate with new features. cgroup v1 is still used in upgraded clusters that have install born dates prior to {product-title} 4.14. cgroup v1 can still be used by changing the `cgroupMode` field of the `Node` object to `v1`.
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -1089,6 +1089,13 @@ With this release, specifying an unhealthy pod eviction policy for pod disruptio
...
+
+For more information, see xref:../nodes/clusters/nodes-cluster-cgroups-2.adoc#nodes-clusters-cgroups-2[Configuring the Linux cgroup version on your nodes ].
```
Comment (adellape) [inline release_notes/ocp-4-14-release-notes.adoc:1097]:
```suggestion
For more information, see xref:../nodes/clusters/nodes-cluster-cgroups-2.adoc#nodes-clusters-cgroups-2[Configuring the Linux cgroup version on your nodes].
```

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (DCChadwick) [reply release_notes/ocp-4-14-release-notes.adoc:1095]: ✓

Comment (DCChadwick) [reply release_notes/ocp-4-14-release-notes.adoc:1095]: ✓

File: release_notes/ocp-4-14-release-notes.adoc (hunk 2 above)
Comment (DCChadwick) [reply release_notes/ocp-4-14-release-notes.adoc:1097]: ✓

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -1089,6 +1089,13 @@ With this release, specifying an unhealthy pod eviction policy for pod disruptio
 
 For more information, see xref:../nodes/pods/nodes-pods-configuring.adoc#pod-disruption-eviction-policy_nodes-pods-configuring[Specifying the eviction policy for unhealthy pods].
 
+[id="ocp-4-14-nodes-cgroupv2-default"]
+==== Linux Control Groups version 2 is now default
+
+Beginning with {product-title} 4.14, new installs use Control Groups version 2 by default, also known as cgroup v2, cgroup2, or cgroupsv2. This enhancement includes many bug fixes, performance improvements, and the ability to integrate with new features. cgroup v1 is still used in upgraded clusters that have initial installation dates prior to {product-title} 4.14. cgroup v1 can still be used by changing the `Node` object to `v1`.
```
Comment (adellape) [inline release_notes/ocp-4-14-release-notes.adoc:1095]: From looking at how things are discussed/marked-up in the `xref`'d section below (https://docs.openshift.com/container-platform/4.14/nodes/clusters/nodes-cluster-cgroups-2.html#nodes-clusters-cgroups-2_nodes-cluster-cgroups-2), it seems like this more correctly should be stated as:
```suggestion
Beginning with {product-title} 4.14, new installs use Control Groups version 2 by default, also known as cgroup v2, cgroup2, or cgroupsv2. This enhancement includes many bug fixes, performance improvements, and the ability to integrate with new features. cgroup v1 is still used in upgraded clusters that have initial installation dates prior to {product-title} 4.14. cgroup v1 can still be used by changing the `cgroupMode` field in the `node.config` object to `v1`.
```

Comment (DCChadwick) [reply release_notes/ocp-4-14-release-notes.adoc]: done

**Review by adellape — APPROVED** [review]

**Pull request merged**